src/components/ExperienceContainer.jsx

```diff
@@ -0,0 +1,54 @@
+import { useState, useEffect } from "react";
+import "./ExperienceContainer.css";
+
+export const ExperienceContainer = () => {
+  const [experience, setExperience] = useState([]);
+
+  useEffect(() => {
+    setExperience([
+      {
+        id: 1,
```

Comment:
nit: can we have these values as consts in the file itself so they can be reused when being used for actual values.

here and in the other files